authorization/src/main/java/com/terabite/chat/controller/ChatController.java

```diff
@@ -25,13 +26,14 @@
 import com.terabite.chat.service.ChatUserService;
 import com.terabite.chat.service.MessageService;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
 
 
 
 @Controller
-@RequestMapping("/v1/chat")
+@RequestMapping("")
```

Comment:
Lets make a note of this. I thin there might be side effects of having this controller as the root endpoint. since "" is the same as "/" I believe. So long as it doesn't break anything else I think its fine though

Reply:
Good call, I removed it and instead just have mapping for individual functions